R/dbplyr.R

```diff
@@ -222,14 +232,6 @@ db_write_table.Pool <- function(con, table, types, values,
                         values = values, temporary = temporary, ...)
 }
 
-#' @rdname dplyr-db-methods
-sql_escape_ident.Pool <- function(con, x) {
-  db_con <- poolCheckout(con)
-  on.exit(poolReturn(db_con))
-  dplyr::check_dbplyr()
-  dplyr::sql_escape_ident(db_con, x = x)
-}
-
```

Comment:
Why isn't this required anymore?

Reply:
Because dbplyr no longer exports this generic, so it doesn't actually do anything. (I think possibly not exporting the generic was a mistake (because I forgot about pool's special needs) but the advantage of the fix here is that there's no need for a dbplyr release)